version/version.go

```diff
@@ -0,0 +1,13 @@
+package version
+
+import "strconv"
+
+type Version struct {
+	Major uint64
+	Minor uint64
+	Patch uint64
+}
+
+func (v *Version) ToString() string {
+	return strconv.FormatUint(v.Major, 10) + "." + strconv.FormatUint(v.Minor, 10) + "." + strconv.FormatUint(v.Patch, 10)
```

Comment:
```suggestion
func (v *Version) String() string {
	return fmt.Sprintf("%d.%d.%d", v.Major, v.Minor, v.Patch)
```

Follow Go standards: rename `ToString()` method used for string transformations to `String()`.

Use `fmt.Sprintf` for the output.